content/python/concepts/functions/terms/arguments/arguments.md

```diff
@@ -0,0 +1,112 @@
+---
+Title: 'Arguments'
+Description: 'Supplies data to a funciton when it is called in a program by order of appearance.'
+Subjects:
+  - 'Computer Science'
+  - 'Data Science'
+Tags:
+  - 'Functions'
+  - 'Parameters'
+  - 'Arguments'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/computer-science'
+---
+
+Unless specified otherwise, arguments passed into a called function are assigned to each parameter in the order in which they appear in the function definition. Thus, they are also known as "positional arguments".
```

Comment:
```suggestion
Arguments supply data to a function when it is called in a program. Unless specified otherwise, arguments passed into a called function are assigned to each parameter in the order in which they appear in the function definition. Thus, they are also known as "positional arguments".
```